services/wallet/responses/router_transactions.go

```diff
@@ -43,7 +43,8 @@ type RouterSentTransaction struct {
 	ToChain     uint64        `json:"toChain"`
 	FromToken   string        `json:"fromToken"`
 	ToToken     string        `json:"toToken"`
-	Amount      string        `json:"amount"` // amount of the transaction
+        Amount      string        `json:"amount"` // amount of the transaction
+        AmountOut      string        `json:"amountOut"` // amount of the transaction sent
```

Comment:
@Khushboo-dev-cpp why do we need the amount out? It is supposed to be used internally only on the statusgo side.

Reply:
Also `RouterSentTransaction` refers to a single tx sent, but `SendDetails` contains info about all general data.

Reply:
Basically `SendDetails` contains global details, what was the intention of the user, and corresponds to the router input params. A single send is composed of one or more transactions, so each `RouterSentTransaction` corresponds to each transaction that was sent. Worth noticing that an approval tx is also a single `RouterSentTransaction`, it means if you want to swap erc20 token you will have 2 `RouterSentTransaction`, one for approval, one for swap.